### PR TITLE
Update bot.py requirements and add support for more environment variables

### PR DESCRIPTION
**Pull request by TheOriginalSoni**
- bot.py dependencies are not all listed, added them to poetry.lock
- Add instructions to env.template
- Add more environment variables to modify for not using si.bitcrafter.net exactly (and similar)
- Remove aioredis from bot.py and add redis as an alternative

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -5,12 +5,11 @@
 import datetime
 import json
 import structlog
-import asyncio
 import async_timeout
-import aioredis
 import re
 from dotenv import load_dotenv
 from PIL import Image
+from redis import asyncio as aioredis
```
Comment (samsullivan) [inline bot.py:12]: I would just leave this package as named `redis` (remove `as aioredis` and replace all other `aioredis` instances with `redis`).

**Review by samsullivan — COMMENTED** [review]
👍

Haven't tested personally, but I did work with you on making all of these changes originally in GCP and they look the same (except cleaner / using env card).

File: bot.py (hunk 1 above)
Comment (TheOriginalSoni) [reply bot.py]: Added new commit to do it

File: bot.py
Comment (TheOriginalSoni) [inline bot.py:1]: Line 106 is still hardcoded 
> r = requests.post(f'http://localhost:8000/api/game/{guid}/link/{after.id}')

**Pull request merged**